src/components/Verification/Verification.tsx

```diff
@@ -246,7 +371,7 @@ const VerificationCred = () => {
 							),
 						},
 						{ data: attribute?.displayName },
-						{
+						!w3cSchema && {
```

Comment:
Why we are using same condition at line 396

Reply:
Both code blocks are different and requires separate conditions.